contents/blog/raising-3m-for-os.md

```diff
@@ -0,0 +1,230 @@
+
+Oh, and you can't pay yourself a huge dividend if you succeed as a way of taking money out of the company, although it might become possible to sell some shares later if you're rocketing (perhaps to help you keep taking risk), and in the short run, you can now probably afford to pay yourself something - hoorah. Going for VC means you are committing to an exit or a failure - you can't really change your mind later that you want to take things more steadily.
+
+Turning your project into a VC backed company means you will have to spend a lot more time dealing with people. Investors, a larger team, probably more customers, and probably a larger community if you succeed in growing. Depending on your personality, this might be a good or a bad thing.
+
+PostHog chose this route. When we're 80, we think we want to have taken a couple of big swings in our lives even if we don't own the whole pie. On the product side, we can build something open source that will outlast us if we get it wrong, and we think the skills and lessons it will give us will help us in our future careers anyway.
+
+## Bootstrapping
+
+By this we mean getting real customers to pay you money and fuelling your own growth at whatever pace you see fit! Basecamp have a great [philosophy](https://basecamp.com/books/calm) on this. 
+
+You get complete control this way. That might make it easier to do the "right thing", or to be more creative in what you build since you won't have to be disrupting or creating a multi billion dollar market as a constraint. However, the financial pressure of payroll in the early days might make it tough to optimize for long run decisions. You need to feed yourself somehow!
+
+We think there is also a lot of risk in this route too. Humans get tired and hungry. It takes a ton of drive to achieve a profitable company. Giving up would be the major risk. We felt this was a similarly high level of risk to inflating your burn rate above your revenue. That risk is likely worse with open source rather than SaaS - you'll have to manage to build a free product first, then a paid version (unless you go down the hosted/not hosted route, which can create competition from the cloud providers).
+
+Tim and I both quit really good jobs to work on PostHog. We felt we could easily have spent 5 years getting back to a similar salary, which a lot more stress and uncertainty.
+
+## Donations
+
+If it's a side project and you have no need to support yourself, this lets you work at your own pace and see if you can make a little money doing it, unless you're exceptionally popular!
+
+Realistically, it will be very hard to support yourself, let alone to grow a team if you want to build a big business and you have rent to pay. However, this could be a cool way to start before doing bootstrapping or VC in the future.
```

Comment:
There's an interesting point around donations: there's not a lot of incentive for companies to donate money, and even if they do the amount will be a fraction on what they would spend on actual software.